packages/suite-web/e2e/tests/onboarding/t1b1-create-wallet.test.ts

```diff
@@ -3,15 +3,16 @@
 
 describe('Onboarding - create wallet', () => {
     beforeEach(() => {
+        cy.task('startEmu', { model: 'T1B1', version: '1-latest', wipe: true });
         cy.task('startBridge');
         cy.viewport(1440, 2560).resetDb();
         cy.prefixedVisit('/');
+        cy.disableFirmwareHashCheck();
```

Comment:
the test is skipped, I tried it locally and it's still broken :x:
Fixing it is out of the scope, but I at least verified that `disableFirmwareHashCheck` is necessary, and that it works (proceed to next parts where the test fails). So when someone wants to fix the test in future, they'll have one less thing to fix :slightly_smiling_face: